express_relay/sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -83,7 +83,7 @@ class SimpleSearcherLimo {
       order.state.outputMint
     );
     const inputAmountDecimals = new Decimal(
-      order.state.remainingInputAmount.toNumber()
+      order.state.initialInputAmount.toNumber()
```

Comment:
maybe the minimum of this value and the `remainingInputAmount` to avoid any issues if the percentage is actually more than what's remaining